client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningClinical.tsx

```diff
@@ -23,6 +25,31 @@ const ContactQuestioningClinical: React.FC<Props> = (props: Props): JSX.Element
 
     const { familyRelationships, interactedContact, updateInteractedContact } = props;
 
+    const [needsToIsolate, setNeedsToIsolate] = useState<boolean>(interactedContact.doesNeedHelpInIsolation);
+
+    React.useEffect(() => {
+        updateInteractedContact(interactedContact, InteractedContactFields.DOES_NEED_ISOLATION, needsToIsolate);
+    },[needsToIsolate]);
+
+    const handleIsolation = (value: boolean) => {
+        value ?
+            Swal.fire({
```

Comment:
I think it's required to change the font of the Swal text as it's done in the rest of the project.

Reply:
תוקן